src/router/Routes.tsx

```diff
@@ -71,13 +75,29 @@ export function Home() {
     );
 }
 
+export function EditProfile() {
+
+    let {userId} = useParams();
```

Comment:
For now and for testing porpuses we can get the user id from the URL. But when the login is finished we should use the user id from another place (could be local storage or create a [react context](https://reactjs.org/docs/context.html))